rviz_default_plugins/src/rviz_default_plugins/displays/depth_cloud/depth_cloud_display.cpp

```diff
@@ -207,7 +207,6 @@ void DepthCloudDisplay::setTopic(const QString & topic, const QString & datatype
     QString transport = topic.mid(index + 1);
     QString base_topic = topic.mid(0, index);
 
-    depth_transport_property_->setString(transport);
     depth_topic_property_->setString(base_topic);
```

Comment:
Actually, isn't this whole block of code wrong?  Detecting the type based on the topic name is always going to fail, so I think we should just replace this entire block with a call to `depth_topic_property_->setString(topic)`.  What do you think?